src/View/Main.java

```diff
@@ -27,7 +27,7 @@
 
 public class Main extends Application {
 	@Override
-	public void start(Stage primaryStage) throws Exception {
+	public void start(Stage primaryStage) throws RuntimeException {
```

Comment:
`RuntimeException` is an unchecked exception type.  I think it better to simply remove the `throws` declaration here.  We can handle the `RuntimeException` as we develop a concept for more comprehensive handling of exceptions and errors.